plugins/com.google.cloud.tools.eclipse.appengine.facets/src/com/google/cloud/tools/eclipse/appengine/facets/FacetInstallDelegate.java

```diff
@@ -111,16 +112,10 @@ private static void createConfigFiles(IProject project, IProgressMonitor monitor
       configDir = (IFolder) current;
     }
 
-    InputStream in = FacetInstallDelegate.class.getResourceAsStream("templates/" + APPENGINE_WEB_XML + ".ftl");
-    if (in == null) {
-      IStatus status = StatusUtil.error(FacetInstallDelegate.class,
-          "Could not load template for " + APPENGINE_WEB_XML);
-      throw new CoreException(status);
-    }
-
-    IFile configFile = configDir.getFile(APPENGINE_WEB_XML);
-    if (!configFile.exists()) {
-      configFile.create(in, true, monitor);
-    }
+    appEngineWebXml.create(new ByteArrayInputStream(new byte[0]), true, monitor);
```

Comment:
What's the rationale of the empty file?  Would be useful to see in a comment.  Is it to ensure Eclipse sees the file?  If so, you should execute `appEngineWebXml.refreshLocal(IResource.DEPTH_ZERO, monitor)`.